src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -283,16 +298,36 @@ void ThreadStateBar::DoUpdatePrimitives(PrimitiveAssembler& primitive_assembler,
       });
 }
 
+std::optional<ThreadStateSliceInfo> ThreadStateBar::FindSliceFromScreenCoords(float x,
```

Comment:
In general we should work with world coordinates in CaptureViewElements. We had some legacy code (f.e. OnPick) that it's still  using screen_coords but it should be the exception. And MouseEvents is using world coordinates, so this is wrong. 

Please use world_coordinates:
`ThreadStateBar::FindSliceFromWorldCoords(Vec2& pos)`